sim/target-agnostic.mk

```diff
@@ -275,10 +275,12 @@ unittest_args = \
 		CONFIG=$(UNITTEST_CONFIG) \
 		TOP_DIR=$(chipyard_dir)
 
-run-midas-unittests: $(chisel_srcs)
+.PHONY: run-midas-unittests
+run-midas-unittests:
 	$(MAKE) -f $(simif_dir)/unittest/Makefrag $@ $(unittest_args)
 
-run-midas-unittests-debug: $(chisel_srcs)
+.PHONY: run-midas-unittests-debug
+run-midas-unittests-debug:
```

Comment:
These targets look like they were promoted from a subdir to be `.PHONY`  targets of the toplevel makefile.  However, their dependency lists are a variable that isn't defined correctly for the subdir in question and is only defined by the TARGET_PROJECT_MAKEFRAGS for midasexamples and fasedtests.

I think the correct thing here is to make them explicitly `.PHONY` and remove their empty (or incorrect) dependency lists.